### DISCUSSION TITLE
Is there any docker image for console lambda?

### DISCUSSION THREAD

**Discussion by pekhota** [brefphp/bref#1087] [Support]
I've found bref/php-80-fpm for web lambda apps and bref/php-80 as a base, but there is no docker image for console app. Is there any instruction how to create console docker app?

**Comment by deleugpn** [comment]
The `function` layer (bref/php-80) is a PHP CLI execution that can be used as a console execution

**Reply by mnapoli** [reply]
👍  this is documented here: https://bref.sh/docs/web-apps/local-development.html#console-applications

**Reply by pekhota** [reply]
I want to deploy into lambda an image with an ability to run `php artisan` command in the same way as we can do with bref console layer.

**Comment by pekhota** [comment]
It works for local development but you can't deploy that into lambda as image. Here is my solution:
1. Download the content of console layer. 
2. Create Dockerfile with the following content:
```
FROM bref/php-80

COPY . /var/task
COPY ./bref/console/bootstrap /opt/bref/bootstrap.php
COPY ./bref/console/breftoolbox.php /opt/bref/breftoolbox.php

CMD ["artisan"]
```
Where ./ is the path to your app (ex. laravel) and ./bref/console/ is the path to console layer content.

You can download console layer with the following command:
```
URL=$(aws lambda get-layer-version-by-arn --arn arn:aws:lambda:us-east-1:209497400698:layer:php-80:23 --query Content.Location --output text)
curl ${URL} -o console.zip
```

**Reply by pekhota** [reply]
Then you can build your dockerfile, publish it to ecr and deploy into lambda. 
After that the command `./vendor/bin/bref cli your_lambda --  ` will work.
